docker-compose.yml

```diff
@@ -5,10 +5,7 @@ services:
     image: mariadb
     restart: unless-stopped
     environment:
-      - MYSQL_DATABASE=csa1
-      - MYSQL_PASSWORD=thisDefaultUserPasswordMustBeChanedInProduction
       - MYSQL_ROOT_PASSWORD=ofCourseThisIsTrueForTheRootPasswordToo
```

Comment:
Actually without the value of this property: `MYSQL_PASSWORD=thisDefaultUserPasswordMustBeChanedInProduction` the Root password doesn't make sense anymore....

Reply:
The database user, password and name is now in the config/db/db_schema.sql. The root stays here because in case of having more than one instance, the root should be able to access to all dbs.